### PR TITLE
ci: Pipeline files

### PR DESCRIPTION
**Pull request by Lee31416**
GitHub Issue: #

## Proposed Changes

 - [ ] Bug fix
 - [ ] Feature
 - [ ] Code style update (formatting)
 - [ ] Refactoring (no functional changes, no api changes)
 - [x] Build or CI related changes
 - [ ] Documentation content changes
 - [ ] Other, please describe:

## What is the current behavior?

No pipelines files.

## What is the new behavior?

Added pipeline files

## Checklist

Please check that your PR fulfills the following requirements:

- [ ] Documentation has been added/updated.
- [ ] Automated tests for the changes have been added/updated.
- [ ] Updated [BREAKING_CHANGES.md](../BREAKING_CHANGES.md) (if you introduced a breaking change).

### CODE REVIEW

File: src/ReviewService.Abstractions/ReviewService.Abstractions.csproj [hunk 1]
```diff
@@ -11,5 +11,6 @@
 		<PackageProjectUrl>https://github.com/nventive/ReviewService</PackageProjectUrl>
 		<GenerateDocumentationFile>true</GenerateDocumentationFile>
 		<PackageTags>review;maui;winui;ios;android</PackageTags>
+		<Configurations>Debug;Release;NuGet</Configurations>
```
Comment (Soap-141) [inline src/ReviewService.Abstractions/ReviewService.Abstractions.csproj]: We forgot to remove this.

File: src/ReviewService/ReviewService.csproj [hunk 2]
```diff
@@ -11,6 +11,7 @@
 		<PackageProjectUrl>https://github.com/nventive/ReviewService</PackageProjectUrl>
 		<GenerateDocumentationFile>true</GenerateDocumentationFile>
 		<PackageTags>review;maui;winui;ios;android</PackageTags>
+		<Configurations>Debug;Release;NuGet</Configurations>
```
Comment (Soap-141) [inline src/ReviewService/ReviewService.csproj]: We forgot to remove this.

**Review by Soap-141 — CHANGES_REQUESTED** [review]

**Review by Soap-141 — APPROVED** [review]

File: build/gitversion.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+assembly-versioning-scheme: MajorMinorPatch
+mode: ContinuousDeployment
```
Comment (takla21) [inline build/gitversion.yml:2]: Don't we want to use proper verisionning (similar to chinook packages) 

references: https://github.com/nventive/Chinook.Navigation/blob/main/build/gitversion.yml

Comment (takla21) [reply build/gitversion.yml:2]: CC @jeanplevesque

**Review by takla21 — APPROVED** [review]

File: build/gitversion.yml (hunk 3 above)
Comment (Soap-141) [reply build/gitversion.yml:2]: We could, we are based on `BiometryService` and other similar packages.

Comment (jeanplevesque) [reply build/gitversion.yml:2]: Use this for now and we'll switch it once we have a first stable version.

File: build/azure-pipelines.yml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    - name: PackageOutputPath # Path where nuget packages will be copied to.
+      value: $(Build.ArtifactStagingDirectory)
+
+    workspace:
+      clean: all # Cleanup the workspace before starting.
+
+    steps:
+    - template: stage-build.yml
+
+- stage: Release
```
Comment (jeanplevesque) [inline build/azure-pipelines.yml]: Disable this stage for now to avoid pushing to nuget.org too soon.

File: build/gitversion.yml (hunk 3 above)
Comment (Soap-141) [reply build/gitversion.yml:2]: Keep it like this for now until we have a stable version.

**Pull request merged**